app/assets/stylesheets/mobile/discourse.scss

```diff
@@ -9,16 +9,14 @@ body {
   background-color: var(--secondary);
 }
 
-.ios-device {
-  textarea {
-    background-color: var(--secondary);
-    font-size: var(--font-size-ios-input);
-    -webkit-tap-highlight-color: transparent;
-  }
+textarea {
+  background-color: var(--secondary); // TODO: why was this iOS-only
```

Comment:
Need to test with/without this on a few browsers to work out why 👀

Reply:
I think this might have been a mistake? see https://github.com/discourse/discourse/pull/7480/files#diff-ba31ce59ea78b5348c6186b59600316cd0922365355d867c63b7de632e1920ca

it was previously applied on all mobile browsers, and that PR didn't justify the change

Reply:
Ok, dropping it